AnimView/frmSprites.cpp

```diff
@@ -113,34 +148,40 @@ void frmSprites::_onNext(wxCommandEvent& evt)
                 break;
             }
         }
-        if(::wxFileExists(s))
-        {
+        if (::wxFileExists(s)) {
             m_txtTable->SetValue(s);
             return;
         }
     }
 }
 
-void frmSprites::load(bool bComplex)
-{
-    if(!m_oAnims.loadTableFile(m_txtTable->GetValue())
-     ||!m_oAnims.loadSpriteFile(m_txtData->GetValue().IsEmpty() ? m_txtTable->GetValue().BeforeLast('.')+L".DAT" : m_txtData->GetValue())
-     ||!m_oAnims.loadPaletteFile(m_txtPalette->GetValue()))
-    {
+void frmSprites::load(bool bComplex) {
+    if (!m_oAnims.loadTableFile(m_txtTable->GetValue()) ||
+        !m_oAnims.loadSpriteFile(
+                m_txtData->GetValue().IsEmpty()
+                        ? m_txtTable->GetValue().BeforeLast('.') + L".DAT"
+                        : m_txtData->GetValue()) ||
+        !m_oAnims.loadPaletteFile(m_txtPalette->GetValue())) {
```

Comment:
Indentation of this line versus the next one is wrong imho. There should always be a distinction between code in the condition and code in the 'then' part.

Reply:
This example has a condition that's far too long to be sane, it should be split up and put in it's own function.

I agree, jquery/eslint a style that would require or allow a blank line at the top of the code in the condition in this case would be better. In the end I opted to just keep a fairly minimal clang-format based on google's style, and that doesn't allow the blank line here.

Rather than fight it, shorten the conditional so it's not confusing.